AmpleOrganics/Blaze.vue
# Docs button

This is the button for docs, and full setup of having our components readily available for creation

this is the more techincal portion of this and not the final result for v1, this will go through the tech writers at some point but the goal is to the function of the docs themselves eg routing and dynamic prop changing

this has some really gross unescessary looking scss stuff, but with this version of vuepress it seems necessary afaik, i will in the near future to cleaning it up. cssnano(built into vue-cli) handles all duplication really well so im not too worried in the meantime


# QA 
pull the repo
run ``` yarn && yarn docs:dev``` to spin up the server
click components and navigate to button
make sure allt he selects/inputs and checkboxes affect the component, and make sure all the props are accounted for (except formName? gonna include that in a future form building advanced usage kind of thing)
